modyn/supervisor/internal/grpc_handler.py

```diff
@@ -276,69 +278,47 @@ def start_training(
             optimizer_config["source"] = optimizer["source"]
             optimizer_config["param_groups"] = []
             for param_group in optimizer["param_groups"]:
-                config_dict = param_group["config"] if "config" in param_group else {}
+                config_dict = param_group.get("config") or {}
```

Comment:
`get` has the second parameter which is returned if key is absent

Reply:
I know, the two variants are not equivalent though and unfortunately, the key absence isn't enough here:

`{"key": None}.get("key", {}) == None`
`{"key": None}.get("key") or {} == {}`